app/views/users/registrations/new.html.erb

```diff
@@ -0,0 +1,40 @@
+<div class="w-screen h-screen flex">
+
+    <div class="max-w-[600px] max-h-[650px] w-full h-full m-auto rounded rounded-lg shadow-[0_0_20px_0_rgba(0,0,0,0.15)] flex flex-wrap">
+    <h3 class="m-6 text-xl leading-5 font-semibold">Step 2 of 3</h3>
+    <%= form_for(resource, as: resource_name, url: registration_path(resource_name), html: { class: 'w-full mx-14 flex flex-col gap-y-5' }) do |f| %>
+        <%#= image_tag 'twitter_icon', class:'w-8 h-8 mx-auto mb-2' %>
+        <%= render '/shared/errors', resource: %>
+
+        <h2 class="text-2xl font-semibold text-black mb-2.5">Create your account</h2>
+
+        <div class="field w-full">
+        <%= f.text_field :name, autofocus: true, autocomplete: "name", placeholder: 'Name', class: 'w-full py-5 px-6 text-xs rounded-md border border-[#D9D9D9]' %>
+        </div>
+
+        <div class="w-full">
+        <%= f.email_field :email, autocomplete: "email", placeholder: 'Email', class: 'w-full py-5 px-6 text-xs rounded-md border border-[#D9D9D9]' %>
+        </div>
+        
+        <div class="field w-full">
+        <%= f.password_field :password, autocomplete: "password", placeholder: 'Password', class: 'w-full py-5 px-6 text-xs rounded-md  border border-[#D9D9D9]' %>
+        </div>
+
+        <div>
+            <h4 class="font-semibold text-base my-2.5" >Date of birth</h4>
```

Comment:
maybe create a `label` tailwind class too, so you can apply that to all the labels, and have them be consistent